server/src/main/kotlin/suwayomi/tachidesk/opds/impl/Opds.kt

```diff
@@ -102,14 +105,14 @@ object Opds {
                                 conditions += (MangaTable.title.lowerCase() like "%${title.lowercase()}%")
                             }
 
-                            if (conditions.isEmpty()) (MangaTable.inLibrary eq true) else conditions.reduce { acc, op -> acc and op }
+                            conditions.reduce { acc, op -> acc and op }
                         }.groupBy(MangaTable.id)
                         .orderBy(MangaTable.title to SortOrder.ASC)
                 val totalCount = query.count()
                 val mangas =
                     query
-                        .limit(ITEMS_PER_PAGE)
-                        .offset(((pageNum - 1) * ITEMS_PER_PAGE).toLong())
+                        .limit(serverConfig.opdsItemsPerPage.value)
```

Comment:
Add this to every use of opdsItemsPerPage
```suggestion
                        .limit(serverConfig.opdsItemsPerPage.value.coerceIn(10, 5000))
```

Reply:
Added. Let me know if it’s not the right way, still getting used to Kotlin and all...

Reply:
Looks good